### DISCUSSION TITLE
Exclude specific reviewer

### DISCUSSION THREAD

**Discussion by belek** [renovatebot/renovate#28318] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

BitBucket

### Please tell us more about your question or problem

Currently my renovate configuration does not include reviewers setting. It creates PR with default reviewers. Howerver I need to exclude one of the default reviewers from PRs generated by Renovate. Is there a way to configure this?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Renovate itself doesn't add any reviewers to PRs by default. Do you mean that Bitbucket is adding them?

**Reply by belek** [reply]
Yes, Bitbucket

**Reply by rarkins** [reply]
How would Renovate stop Bitbucket adding reviewers?

**Reply by belek** [reply]
I do not know that, that's why I asked. I though may be there is some configuration trick to exclude specific reviewer for PRs.

**Reply by belek** [reply]
What if every repository would have specific reviewers listed in renovate configs, would BitBucket still add default ones too?

**Reply by rarkins** [reply, marked as answer]
I think you need to be asking these questions in Bitbucket forums instead- renovate isn't controlling these
